src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay.mdx

```diff
@@ -277,21 +280,31 @@ Here's an example of adding the `crossorigin="anonymous"` attribute:
 
-If needed, you can programmatically invoke a session replay using API methods. Here are some use cases:
+If needed, you can programmatically `force` a session replay to start or stop recording using API methods. Here are some use cases:
 
-* Allow users to opt in for interaction recording. For example, if you have a pop-up asking "Allow this session to be recorded for performance analysis?":
-    1. Upon user consent, call `NREM.startSessionReplay` to initiate recording. 
-    2. Once the session is complete or consent is withdrawn, use `NREM.stopSessionReplay` to stop capturing data.
-* Record sessions based on specific criteria. For example:
+* Allow users to opt in for interaction recording. For example, if you have a pop-up asking "Allow this session to be recorded for performance analysis?" and a user consents:
+    1. Call `newrelic.recordReplay()` to initiate recording. 
+    2. Once the session is complete or consent is withdrawn, use `newrelic.pauseReplay()` to stop capturing data.
+* Record sessions based on specific criteria, such as:
     * Start recording only for sessions on specific URLs such as a product checkout flow.
     * Stop recording for sensitive pages such as `/billing` or `/health-records`.
 
 See the API documentation for instructions:
 
-* [`NREM.startSessionReplay`](/docs/browser/new-relic-browser/browser-apis/start/)
-* [`NREM.stopSessionReplay`](/docs/browser/new-relic-browser/browser-apis/stop/)
+* [`newrelic.recordReplay`](/docs/browser/new-relic-browser/browser-apis/recordReplay/)
+* [`newrelic.pauseReplay`](/docs/browser/new-relic-browser/browser-apis/pauseReplay/)
 
-To ensure all user interactions are captured during manual recording, set sampling rates to 0% in your application settings.
+## Disable automatic session replay tracking [#auto-replays]
+
+You can prevent session replays from automatically starting, even when sampled, by setting API method `autoStart` configuration option to `false`. This is useful in scenarios where you need to prioritize specific actions over automatic sampling. For example, you might not want session replays to automatically run on sensitive pages, like HIPAA-compliant ones. In such cases, you can use the `.start()` API to manually trigger the replay feature after user consent is obtained (and your app calls `.start()`)
+
+Setting `autoStart: false` instructs the agent to defer its initialization until explicitly called using `.start()`. This ensures that session replays will not be created at all, even if the backend samples your application for recording.
+
+<Callout variant="important">
+  Calling `.start()` does not guarantee that a session replay will be recorded. It simply delays the module import and evaluation of sampling results until this point. If you need to force a recording, see [Manually record session replays](#manual-replays).
+</Callout>
+
+To start using this API, see [`newrelic.start`](/docs/browser/new-relic-browser/browser-apis/start/).
```

Comment:
@metal-messiah thanks for your response. Can you review my revision on this section and let me know if it looks good?